DigitalOffice/Services/UserService/Validators/EditUserRequestValidator.cs

```diff
@@ -0,0 +1,52 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.UserService.RestRequests;
+
+namespace LT.DigitalOffice.UserService.Validators
+{
+    public class EditUserRequestValidator : AbstractValidator<EditUserRequest>
+    {
+        public EditUserRequestValidator()
```

Comment:
I think it's wrong to force the frontend to fill in all fields when editing.

Reply:
Hmm, I thought in our previous discussion we decided not to change it in case of potentially existing another solution.